frontend/app/globals.css

```
@@ -30,3 +30,110 @@ body {
   background: #09090b;
   color: #fafafa;
 }
+
+/* Custom Shepherd Theme */
+.shepherd-theme-custom {
+  background: rgba(255, 255, 255, 0.95) !important;
+  backdrop-filter: blur(12px) !important;
+  border: 1px solid rgba(0, 0, 0, 0.15) !important;
+  border-radius: 16px !important;
+  box-shadow: 0 20px 40px -10px rgba(0, 0, 0, 0.1) !important;
+  font-family: 'Inter', sans-serif !important;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)